app/Models/Transaction.php

```diff
@@ -129,6 +130,16 @@ public function recipient(): BelongsTo
         return $this->belongsTo(Wallet::class, 'recipient_id', 'address');
     }
 
+    /**
+     * Get migrated transactions.
+     *
+     * @return Builder
+     */
+    public function scopeMigrated(): Builder
+    {
+        return self::where('recipient_id', config('explorer.migration_address'));
```

Comment:
only `transfer` transaction should be included, currently it includes all types with that recipient address